src/code/code_800F9280.c

```diff
@@ -81,12 +149,6 @@ typedef enum {
     CMDF
 } SeqCmdType;
 
-typedef struct {
-    u8 unk_0;
-    u8 unk_1; // importance?
-} Struct_8016E320;
-extern Struct_8016E320 D_8016E320[][5]; // sorted by unk_1, descending
```

Comment:
this comment was lost, idk if it matters

Reply:
I have this struct documented locally that will be submitted 2 audio PRs from now, so it's not that big of a deal I don't think